content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -31,6 +32,8 @@ The CertificateSigningRequest object includes a PEM-encoded PKCS#10 signing requ
 the `spec.request` field. The CertificateSigningRequest denotes the signer (the
 recipient that the request is being made to) using the `spec.signerName` field.
 Note that `spec.signerName` is a required key after API version `certificates.k8s.io/v1`.
+The client may optionally set the `spec.durationHint` to request a particular
```

Comment:
It seems like this needs to change to reflect the field renaming to `.spec.expirationSeconds`